pkg/config/config.go

```diff
@@ -124,3 +124,17 @@ type Config struct {
 	SubscriptionID string
 	ResourceGroup  string
 }
+
+type CertificateConfig struct {
+	// CAs
```

Comment:
There are more certificates than just the CAs. Also there are the kubeconfigs, do we want these in this struct as well?

Reply:
Sorry, just read your PR comment.